exampleCourse/courseInstances/SectionA/assessments/gallery/questions/infoAssessment.json

```
@@ -28,5 +28,5 @@
             ]
         }
     ],
-    "text": "<p><strong>Comments about this assessment:</strong> </p><p> This assessment includes examples from the question gallery at <a href=https://www.prairielearn.com/gallery>https://www.prairielearn.com</a>.</p>."
+    "text": "<p><strong>Comments about this assessment:</strong> </p><p> This assessment includes examples from the question gallery at <a href=https://www.prairielearn.com/gallery>https://www.prairielearn.com</a>.</p>"
```

Should the link text match the link href?

---

Probably! I also updated the link to fix a 404.

```suggestion
    "text": "<p><strong>Comments about this assessment:</strong> </p><p> This assessment includes examples from the question gallery at <a href=\"https://www.prairielearn.com/catalog/questions\">https://www.prairielearn.com/catalog/questions</a>.</p>"
```